SummaryJavadoc/Example6/config.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+        "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<module name = "Checker">
+    <property name="charset" value="UTF-8"/>
+
+    <!-- do not change severity to 'error', as that will hide errors caused by exceptions -->
+    <property name="severity" value="warning"/>
+
+    <!-- haltOnException is required for exception fixes and reporting of all exceptions -->
+    <property name="haltOnException" value="false"/>
+
+    <!-- BeforeExecutionFileFilters is required for sources of java9 -->
+    <module name="BeforeExecutionExclusionFileFilter">
+        <property name="fileNamePattern" value="module\-info\.java$" />
+    </module>
+
+    <module name="TreeWalker">
+        <module name="SummaryJavadocCheck">
+            <property name="period" value="。"/>
```

Comment:
To generate in good way for this Check we need extractor to have some yml files where we can define what projects should be used at SummaryJavadoc/Example6/list-of-projects.properties .
No reason to run on all this config, 
```
# japan language
WxJava|
```
Should be used.
Let's create separate issue on this and do introduction of config and loading bof it and generation based on it.